in_toto/runlib.py

```diff
@@ -319,7 +319,7 @@ def record_artifacts_as_dict(artifacts, exclude_patterns=None,
 
   return artifacts_dict
 
-def execute_link(link_cmd_args, record_streams):
+def execute_link(link_cmd_args, record_streams, quiet=True):
```

Comment:
I think `quiet` could be mandatory here. `execute_link` is more or less internal API and only called from `in_toto_run`. For `in_toto_run` OTOH `quiet` should indeed be optional.